reactioncommerce/reaction-next-starterkit
# feat: Add and hookup checkout summary to checkout page

Resolves #208
Impact: **minor**
Type: **feature**

## Issue

Add and hookup a cart summary component for the cart page.

## Solution

Using the cart page as a base, condense into a single column layout and with other adjustments to transform it into a cart summary to live in the trailing column of the checkout page.

## Testing
1.  Add items to the cart
2. Go to checkout page
3. Verify that the summary has the proper items and totals
